c/weld.h

```diff
@@ -9,6 +9,18 @@
 
 #include <stdint.h>
 
+// Log level constants -- should match WeldLogLevel.
+
+/** A Weld log level. */
+typedef uint64_t weld_log_level_t;
+
+weld_log_level_t WELD_LOG_LEVEL_OFF = 0;
```

Comment:
Hmm yeah, I don't think that can be guaranteed. You could make this an enum and then cast the value you get passed from rust to the enum type, but maybe this is better then.